tests/Kerbero.WebApi.Tests/ModelStateDictionaryExtensionsTest.cs

```diff
@@ -31,7 +31,7 @@ public void AddErrorAndReturnAction_Test(KerberoError error)
 		// Arrange
 		
 		// Act
-		var action = ModelState.AddErrorAndReturnAction(error) as ObjectResult;
+		var action = ModelState.AddErrorAndReturnAction(error) as ObjectResult ?? throw new NullReferenceException();
```

Comment:
Why would we want to throw inside a test?

Reply:
The action? does the same thing. If it is null the test assert.